build.gradle.kts

```
@@ -58,8 +75,20 @@ configurations.all {
 dependencies {
   implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlinVersion")
   implementation("org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion")
-  implementation("com.github.JetBrains.projector-server:projector-server:$projectorServerVersion")
-
+  when (projectorServerSource) {
+    "space" -> {
+      implementation("org.jetbrains.projector:projector-server:$projectorServerVersion")
+    }
+    "jitpack" -> {
+      implementation("com.github.JetBrains.projector-server:projector-server:$projectorServerVersion")
+    }
+    "" -> {
+      throw Exception("projectorServerSource is empty")
+    }
+    else -> {
+      implementation("com.github.JetBrains.projector-server:projector-server:-SNAPSHOT")
+    }
```

Minor suggestion. This piece can be made less error-prone:

```suggestion
    "local", "null" -> {
      implementation("com.github.JetBrains.projector-server:projector-server:-SNAPSHOT")
    }
    else -> {
      throw IllegalStateException("Unknown projectorServerSource = '$projectorServerSource'")
    }
```

So we explicitly support the cases when `local.properties` contain `local` and when `local.properties` are missing (the value will be `null` in that case). These are correct cases, so we use "com.github.JetBrains.projector-server:projector-server:-SNAPSHOT".

And all other cases are unknown (including the case when there is an empty value), so we throw an exception. I've changed its type to IllegalStateException just to reflect the situation more specifically.

If we apply this change, we make the buildscript more foolproof.